frontend/src/components/ContainerComponent.vue

```diff
@@ -1,7 +1,7 @@
 <!-- Main component for seperating semantic contents -->
 <template>
   <div
-    class="p-2 border-1 rounded-md bg-container border-container-border box-border shadow overflow-hidden flex flex-col max-h-0 min-h-full"
```

Comment:
was the component tested after this was removed?

Reply:
Wasn't this your change? None the less no bugs have occured so far.

Reply:
I cant remember. If it neds to be cahnged back somewhere we will just do so in that PR